src/repositories_rust.tf

```diff
@@ -274,6 +274,12 @@ locals {
       ".github/workflows/autosquash.yml" = {
         content = file("templates/rust-all/.github/workflows/autosquash.yml")
       }
+      ".github/workflows/release_checklist.yml" = {
+        content = file("templates/rust-all/.github/workflows/release_checklist.yml")
+      }
+      "scripts/release_checklist.py" = {
```

Comment:
This should be added to the arrow-sanitychecks docker instead

Reply:
https://github.com/Arrow-air/tf-github/pull/140/commits/4eb31b9268274ae2bca55fc48543d26edcf6324f